mysql-test/suite/rocksdb/t/bulk_load_errors.test

```diff
@@ -0,0 +1,39 @@
+--source include/have_rocksdb.inc
+
+CREATE TABLE t1(pk INT, PRIMARY KEY(pk)) ENGINE=RocksDB;
+
+# Make sure we get an error with out of order keys during bulk load
+SET rocksdb_bulk_load=1;
+INSERT INTO t1 VALUES(10);
+INSERT INTO t1 VALUES(11);
+--error ER_KEYS_OUT_OF_ORDER
+INSERT INTO t1 VALUES(9);
+SET rocksdb_bulk_load=0;
+
+# Make sure only 10 and 11 got into the table
+SELECT * FROM t1;
+
+# Make sure we get an error with overlapping data
+SET rocksdb_bulk_load=1;
+INSERT INTO t1 VALUES(1);
+INSERT INTO t1 VALUES(2);
+INSERT INTO t1 VALUES(20);
+INSERT INTO t1 VALUES(21);
+
+# This last crashes the server (intentionally) because we can't return any
+# error information from a SET <variable>=<value>
+--exec echo "wait" > $MYSQLTEST_VARDIR/tmp/mysqld.1.expect
+--error 2013 # CR_SERVER_LOST
+SET rocksdb_bulk_load=0;
+
+--exec grep "RocksDB: Error 197 finalizing last SST file while setting bulk loading variable" $MYSQLTEST_VARDIR/log/mysqld.1.err | cut -d] -f2
+--exec echo "" >$MYSQLTEST_VARDIR/log/mysqld.1.err
```

Comment:
This ruins troubleshooting, and CURRENT_TEST markers in the log which MTR uses. Please restart the server with custom error log location at the start of the testcase, restart with default at the end, and remove the custom error log.